dom/src/main/scala-2/fs2/dom/Dom.scala

```diff
@@ -112,6 +112,40 @@ object Element {
 }
 
 trait HtmlElement[F[_]] extends Element[F]
+object HtmlElement {
+
+  implicit def ops[F[_]](element: HtmlElement[F]): Ops[F] =
+    new Ops(element.asInstanceOf[dom.HTMLElement])
+
+  final class Ops[F[_]] private[HtmlElement] (private val element: dom.HTMLElement) extends AnyVal {
+    def focus(implicit F: Dom[F]): F[Unit] = F.delay(element.focus())
+
+    // facade for overload missing from org.scalajs.dom
+    def focus(options: FocusOptions)(implicit F: Dom[F]): F[Unit] =
+      F.delay {
+        element.asInstanceOf[js.Dynamic].focus(FocusOptions.toJS(options))
+        ()
+      }
+
+    def blur(implicit F: Dom[F]): F[Unit] = F.delay(element.blur())
+
+    def click(implicit F: Dom[F]): F[Unit] = F.delay(element.click())
+
+    def offsetHeight(implicit F: Dom[F]): F[Double] = F.delay(element.offsetHeight)
+
+    def offsetWidth(implicit F: Dom[F]): F[Double] = F.delay(element.offsetWidth)
```

Comment:
Hmm, I don't think these are ever fractional. So we should either use `Int` or `Long` depending on the possible range of values.

> Note: This property will round the value to an integer.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/offsetHeight

Reply:
This is what scala-js-dom given us currently. Do you want to coerce to Int here or fix upstream?

Reply:
Good question. So scala-js-dom is a bit more constrained than us: because it's facades, it's only allowed to use native JS types (so, no `Long`). A lot of time it uses `Double` because that is the canonical representation of a JS number.

Fortunately, we do not have that constraint. So we can determine for ourselves the appropriate, idiomatic wrapper type, and then coerce to that.